python/apps/taiga/src/taiga/stories/stories/api/validators.py

```diff
@@ -31,6 +32,14 @@ class UpdateStoryValidator(BaseModel):
     status: B64UUID | None
     workflow: str | None
 
+    @root_validator
+    def status_or_workflow(cls, values: dict[Any, Any]) -> dict[Any, Any]:
+        status = values.get("status")
+        workflow = values.get("workflow")
+        workflow_has_value = workflow is not None and workflow != ""
+        assert not (status and workflow_has_value), "It's not allowed to update both the status and workspace"
```

Comment:
```python
workflow_has_value = workflow is not None and workflow != ""
assert not (status and workflow_has_value), "It's not allowed to update both the status and workspace"
```
is the same as

```python
assert not (status and workflow), "It's not allowed to update both the status and workspace"
```

Because

```python
>>> bool(None)
False
>>> bool("")
False
>>> bool("text")
True
```

Reply:
I'll change it as it's much clear with the same behaviour. Thanks